### DISCUSSION TITLE
[Feature request] Custom alias creation rules

### DISCUSSION THREAD

**Discussion by grzegor** [anonaddy/anonaddy#371] [Feature Requests]
It would be great if user could provide their own dictionaries, let's say for example `colors.txt` and `fruits.txt`.
Then customize alias creation with them, like: `${colors}_${fruits}` - this would result in picking a random word from colors dict (let's say "red") and random word from fruits dict (let's say "banana"), resuling in someting like `red_banana@mydomain.tld` alias. It's ok if it worked only on custom (user provided) domain names, because of the potential lack of global naming variablity and even on the pro plan since it seems like a very specific feature and would require hosting user's dictionaries on the server.

Also thank you for creating this awesome service!

**Comment by willbrowningme** [comment]
Hmm, this is a good idea but could result in lots of storage depending on max size and whether the lists would be stored as a file or in the database. It may be something I'll only add to the self-hosted version.
